modules/ROOT/pages/type-definitions/directives/index.adoc

```diff
@@ -71,13 +71,13 @@ to populate fields which have not been provided within the input.
 | xref::/schema-configuration/field-configuration.adoc#_relationship[`@relationship`]
 | Configure  xref::/type-definitions/types/relationships.adoc[relationships] between object types.
 
-| *`@relationshipProperties`*
+| xref::/type-definitions/types/relationships/#_relationship_properties[`@relationshipProperties`]
```

Comment:
```suggestion
| xref::/type-definitions/types/relationships.adoc#_relationship_properties[`@relationshipProperties`]
```

Reply:
double-checked locally - this causes an error otherwise:
```
ERROR (asciidoctor): target of xref not found: :/type-definitions/types/relationships/.adoc#_relationship_properties
``

Reply:
Ohh nice find!